vertical-pod-autoscaler/pkg/utils/metrics/quality/quality.go

```diff
@@ -30,6 +33,12 @@ const (
 	metricsNamespace = metrics.TopMetricsNamespace + "quality"
 )
 
+var (
+	// Both buckets are of length 17, 35 with prepended negative buckets
```

Comment:
I don't understand the comment, why are there two numbers?

Reply:
Removed altogether.